frontend/src/components/Asset.svelte

```diff
@@ -0,0 +1,72 @@
+<script lang="ts">
+    function round(value: number): number {
```

Comment:
Could instead use `.toFixed(2)` for rounding to two dp?

Reply:
Ohh didn't know that before